OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -79,6 +82,14 @@ public Shroud(Actor self)
 			explored = new CellLayer<bool>(map);
 		}
 
+		void INotifyCreated.Created(Actor self)
+		{
+			fogEnabled = self.World.LobbyInfo.GlobalSettings.Fog;
+			var shroudEnabled = self.World.LobbyInfo.GlobalSettings.Shroud;
+			if (!shroudEnabled)
```

Comment:
The final implementation of this line (after `GlobalSettings.Shroud` is removed) is too long to put inside the if clause.